docker/mariadb_tests/Dockerfile

```diff
@@ -13,6 +13,8 @@ COPY cmd/ cmd/
 COPY main/ main/
 COPY utility/ utility/
 
+RUN cp -r /go/tmp/brotli/* vendor/github.com/google/brotli
+
```

Comment:
It looks like that it can be done in the same way as for MySQL (check Makefile > mysql_integration_test vs mariadb_integration_test )